src/app/pages/WelcomePage/WelcomePage.module.css

```diff
@@ -0,0 +1,33 @@
+.container {
+  background-image: url('../../components/Welcome/WelcomePageImage.jpg');
```

Comment:
In your [deployment](https://dear-diary-r-welcome-xgtptdftw.herokuapp.com/storybook/?path=/story/pages-welcomepage--default) I can't see the background image - I think this might fix it, but you should check :) 

```suggestion
  background-image: url('../../components/Welcome/welcomePageImage.jpg');
```

Reply:
Thanx! I needed to edit the picture and changed the name to start with a capital letter. I will check the result.